README.md

```diff
@@ -1,31 +1,77 @@
-# README
+<!-- PROJECT SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Issues][issues-shield]][issues-url]
+<!-- PROJECT LOGO -->
+<br/>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<p align="center">
+  <a href="https://github.com/Head-To-Toe/headToToe-be">
+    <img src="https://github.com/Head-To-Toe/headToToe-be/blob/main/docs/head_to_toe_official.png">
+  </a>
+  <p align="center">
+    <a href="https://github.com/Head-To-Toe/headToToe-be/issues">Report Bug</a>
+  </p>
+</p>
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  
+  <ol>
+    <li><a href="#about-the-project">About The Project</a></li>
+    <li><a href="#getting-started">Getting Started</a></li>
+    <li><a href="#endpoints">Endpoints</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#contributors">Contributors</a></li>
+  </ol>
+  
+</details>
+<!-- ABOUT THE PROJECT -->
 
-* Ruby version
+## About The Project
+Head to Toe is a publically sourced LGBTQ+ health professional referral application. Find peer-acknowledged doctor and mental health professionals care across the U.S. Or, submit a resource for review and help others find care near them.
+<br> 
+<br>
+The Head to Toe backend is an API that aggregates doctor and mental health professional data to its partner frontend application. The frontend repo is located [here](https://github.com/Head-To-Toe/headToToe).
```

Comment:
Maybe I am blind but we should include a link to the deployed FE

Reply:
> Maybe I am blind but we should include a link to the deployed FE

Line 36 here covers that :D